ios/FluentUI/Notification/NotificationView.swift

```diff
@@ -361,10 +356,38 @@ open class NotificationView: UIView, FluentUIWindowProvider {
 
     open override func didMoveToWindow() {
         super.didMoveToWindow()
-        tokens.updateForCurrentTheme()
+        updateNotificationTokens()
         updateWindowSpecificColors()
     }
 
+    // MARK: - TokenizedControl
+    public typealias TokenType = NotificationTokens
+    public func overrideTokens(_ tokens: NotificationTokens?) -> Self {
+        overrideTokens = tokens
+        return self
+    }
+
+    var state: NotificationView { self }
+
+    /// Design token set for this control, to use in place of the control's default Fluent tokens.
+    var overrideTokens: NotificationTokens?
+
+    /// Style to draw the control.
+    public var style: MSFNotificationStyle {
+        didSet {
+            tokens.style = style
```

Comment:
Do you need to call `updateForStyle()` here as well?

Reply:
I think it actually only needs to be called in initialize()!